apps/firebase.json

```diff
@@ -5,7 +5,7 @@
   },
   "functions": {
     "predeploy": "bazel build //apps/functions:functions_compiled",
-    "source": "../dist/bin/apps/functions"
+    "source": "../apps/functions/dist/functions"
```

Comment:
```suggestion
    "source": "./functions/dist/functions"
```

Reply:
is a symlink also working on windows? likely it does at this point.

Reply:
I believe it should and will verify.